src/index.js

```diff
@@ -1,5 +1,12 @@
 import transform from './lib/transform';
 
+/**
+ * @param {{precision?: number | false, 
+ *          preserve?: boolean, 
+ *          warnWhenCannotResolve?: boolean, 
+ *          mediaQueries?: boolean, 
+ *          selectors?: boolean}} opts
+ */
```

Comment:
You can use `@typedef` to improve types and allow to import them

Reply:
Done